examples/custom-config/README.md

```diff
@@ -0,0 +1,23 @@
+# Default Example
+
+This is an example of using the Terraform module for Lacework CSPM integration
+using custom parameters.
+
+```hcl
+module "lacework_oci_cfg_integration" {
+  source = "../.."
```

Comment:
You would use this source inside actual code (inside `.tf` files), but on the README we try to use the actual source and version from the registry. Look at other modules for examples https://github.com/lacework/terraform-aws-config/blob/main/examples/default-config/README.md

Reply:
Ah, I see. Fixed it!